wallet/src/de/schildbach/wallet/ui/dashpay/CreateIdentityViewModel.kt

```diff
@@ -56,18 +56,28 @@ class CreateIdentityViewModel @Inject constructor(
             val isRestoring = withContext(Dispatchers.IO) { blockchainIdentityDataDao.get(BlockchainIdentityConfig.RESTORING) ?: false }
 
             if (isRestoring) {
-                val identityId = withContext(Dispatchers.IO) {blockchainIdentityDataDao.get(BlockchainIdentityConfig.IDENTITY_ID) }
+                val identityId = withContext(Dispatchers.IO) { blockchainIdentityDataDao.get(BlockchainIdentityConfig.IDENTITY_ID) }
```

Comment:
`get` is a coroutine that should already be dispatched to IO, so `withContext` might not be needed.

Reply:
I couldn't tell that `get` is dispatched to IO from the code.

Reply:
Yes, `DataStore` is async by default. One of the reasons we're migrating from the blocking `Configuration` class.

Reply:
very good, I will remove the unnecessary `withContext` .  This probably was done in other places as well.